### PR TITLE
Intergrate New Relic into Sim2h Server Docker Box

### PR DESCRIPTION
**Pull request by AshantiMutinta**
## PR summary
This PR sets up the dependancies needed for the new relic to be listening to the sim2h server
## testing/benchmarking notes

( if any manual testing or benchmarking was/should be done, add notes and/or screenshots here )

## followups

( any new tickets/concerns that were discovered or created during this work but aren't in scope for review here )

## changelog

- [ ] if this is a code change that effects some consumer (e.g. zome developers) of holochain core,  then it has been added to [our between-release changelog](https://github.com/holochain/holochain-rust/blob/develop/CHANGELOG-UNRELEASED.md) with the format 

```markdown
- summary of change [PR#1234](https://github.com/holochain/holochain-rust/pull/1234)
```

## documentation

- [ ] this code has been documented according to our [docs checklist](https://hackmd.io/@freesig/Hk9AmKJNS)

### CODE REVIEW

File: default.nix [hunk 1]
```diff
@@ -39,7 +39,11 @@ with holonix.pkgs;
     ''
     ];
 
-  buildInputs = [ holonix.pkgs.libiconv ]
+  # Needed so bindgen can find libclang.so
+  LIBCLANG_PATH="${holonix.pkgs.llvmPackages.libclang}/lib";
+  hardeningDisable = [ "fortify" ];
+
+  buildInputs = [ holonix.pkgs.libiconv holonix.pkgs.pcre holonix.pkgs.cmake holonix.pkgs.clang]
```
Comment (AshantiMutinta) [inline default.nix]: still compiles after removing gblic @thedavidmeister

Comment (thedavidmeister) [reply default.nix]: @AshantiMutinta can we remove all the extra build inputs?

i thought that with `LIBCLANG_PATH` and `hardeningDisable` it should all work without this change here?

Comment (AshantiMutinta) [reply default.nix]: not sure, will try but I remember us incrementally adding these build inputs., had to add pcre cause it was looking for it in the nix-shell but we probably don't need cmake

Comment (AshantiMutinta) [reply default.nix]: Removed the build inputs

Comment (AshantiMutinta) [reply default.nix]: pcre is needed so restored it to the build inputs

**Comment by AshantiMutinta** [comment]
Merged in #2059

**Pull request closed without merging**